examples/grid/rl_grid_direct_mpc_curr_ctr.py

```diff
@@ -30,13 +30,14 @@
 i_ref_dq = Sequence(np.array([0, 0.1, 0.1, 1]),
                     np.array([[1, 0], [1, 0], [0.3, 0.3], [0.3, 0.3]]))
 
+# Define grid parameters
+grid_params = model.grid.RLGridParameters(Vgr_SI=3300,
+                                          fgr_SI=50,
+                                          Rg_SI=0.01815,
+                                          Lg_SI=5.7773e-4,
+                                          base=base)
 # Define system models
-sys = model.grid.RLGrid(Vgr=3300,
-                        fgr=50,
-                        Rg=0.01815,
-                        Lg=5.7773e-4,
-                        base=base,
-                        ig_ref_init=i_ref_dq(0))
+sys = model.grid.RLGrid(par=grid_params, base=base, ig_ref_init=i_ref_dq(0))
 conv = model.conv.Converter(v_dc=5200, nl=3, base=base)
```

Comment:
v_dc is used as a parameter and an attribute in Converter class, should it follow the same definition with RLGridParameters class and defined as v_dc_SI here?

Reply:
Also, if the parameter that are note followed by _SI indicate the pu system, should the base parameters in line 27 be Vgr_B, Igr_B and fgr_B since they are SI quantities?